tests/lib/prepare.sh

```diff
@@ -344,6 +344,104 @@ repack_snapd_snap_with_deb_content() {
     rm -rf "$UNPACK_DIR"
 }
 
+setup_core_for_testing_by_modify_writable() {
+    UNPACK_DIR="$1"
+
+    # create test user and ubuntu user inside the writable partition
+    # so that we can use a stock core in tests
+    mkdir -p /mnt/user-data/test
+
+    # create test user, see the comment in spread.yaml about 12345
+    mkdir -p /mnt/system-data/etc/sudoers.d/
+    echo 'test ALL=(ALL) NOPASSWD:ALL' >> /mnt/system-data/etc/sudoers.d/99-test-user
+    echo 'ubuntu ALL=(ALL) NOPASSWD:ALL' >> /mnt/system-data/etc/sudoers.d/99-ubuntu-user
+    # modify sshd so that we can connect as root
+    mkdir -p /mnt/system-data/etc/ssh
+    cp -a "$UNPACK_DIR"/etc/ssh/* /mnt/system-data/etc/ssh/
+    # core18 is different here than core16
+    sed -i 's/\#\?\(PermitRootLogin\|PasswordAuthentication\)\>.*/\1 yes/' /mnt/system-data/etc/ssh/sshd_config
+    # ensure the setting is correct
+    grep '^PermitRootLogin yes' /mnt/system-data/etc/ssh/sshd_config
+
+    # build the user database - this is complicated because:
+    # - spread on linode wants to login as "root"
```

Comment:
s/linode/google cloud and linode/ ?

Reply:
Yeah, this is just copied from the original text - but it's a bit outdated :)